pandas/tests/test_generic.py

```diff
@@ -996,6 +996,59 @@ def test_describe_percentiles_insert_median(self):
         self.assertTrue('0%' in d1.index)
         self.assertTrue('100%' in d2.index)
 
+    def test_describe_percentiles_unique(self):
+        # GH13104
+        df = tm.makeDataFrame()
+        with self.assertRaises(ValueError):
+            df.describe(percentiles=[0.1, 0.2, 0.4, 0.5, 0.2, 0.6])
+        with self.assertRaises(ValueError):
+            df.describe(percentiles=[0.1, 0.2, 0.4, 0.2, 0.6])
+
+    def test_describe_percentiles_formatting(self):
+        # GH13104
```

Comment:
As for now, I don't have a better idea than to raise an exception if a data frame has no columns:
```python
    def describe(self, percentiles=None, include=None, exclude=None):
        if self.ndim >= 3:
            msg = "describe is not implemented on Panel or PanelND objects."
            raise NotImplementedError(msg)
+       elif self.ndim == 2 and self.columns.size == 0:
+           raise ValueError("Cannot describe a DataFrame without columns")
```

An empty data frame with columns can still be treated as an empty series.